constraints/valid_values/validation_list_timestamp_ranges.isl

```diff
@@ -0,0 +1,18 @@
+type::{
+  valid_values: [
+    range::[min, 2000-01-01T00:00Z],
+    2000-01-11T00:00Z,
+    range::[2000-02-01T00:00Z, max]
+  ],
+}
+valid::[
+  0001-01-01T00:00Z,
+  2000-01-01T00:00Z,
+  2000-02-01T00:00Z,
+  2000-03-01T00:00Z,
+  2000-01-11T00:00Z,
```

Comment:
Could you add a couple of values showing the same instant at a different precision? For example:
```
2000-01-11T
2000-01-11T00:00Z,
2000-01-11T00:00:00Z
2000-01-11T00:00:00.000Z
```